### PR TITLE
Cocoa bindings

### PR DESCRIPTION
**Pull request by michaelfgoldenTwoPG**
Adds simple error handling for null client state.
Hides Xcode warnings from incomplete librpc documentation.

### CODE REVIEW

File: bindings/objective-c/librpc/librpc.m [hunk 1]
```diff
@@ -306,9 +306,14 @@ @implementation RPCClient {
...
+- (BOOL)connect:(NSString *)uri {
     client = rpc_client_create([uri UTF8String], NULL);
-    conn = rpc_client_get_connection(client);
+    if (client) {
+        conn = rpc_client_get_connection(client);
+        return YES;
+    } else {
+        return NO;
+    }
 }
```
Comment (jceel) [inline bindings/objective-c/librpc/librpc.m]: Shouldn't a failed connect throw an exception instead?

Comment (michaelfgoldenTwoPG) [reply bindings/objective-c/librpc/librpc.m]: I need to know if the service is running without crashing. Is there another option for doing that?

Comment (jceel) [reply bindings/objective-c/librpc/librpc.m]: As we discussed, the best option would be to return NSError * (as output parameter). error cause can be obtained with `rpc_get_last_error()`.

Comment (michaelfgoldenTwoPG) [reply bindings/objective-c/librpc/librpc.m]: 👍 I will update the pr

**Pull request merged**